<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabFrame</class>
 <widget class="QWidget" name="TabFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QToolButton" name="delToolButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../property.qrc">
           <normaloff>:/objectcontroller/images/Remove.png</normaloff>:/objectcontroller/images/Remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="upToolButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../property.qrc">
           <normaloff>:/objectcontroller/images/Up.png</normaloff>:/objectcontroller/images/Up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="downToolButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../property.qrc">
           <normaloff>:/objectcontroller/images/Down.png</normaloff>:/objectcontroller/images/Down.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="positionSpinBox">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>2147483647.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeComboBox">
         <item>
          <property name="text">
           <string>LeftTab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RightTab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CenterTab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DelimiterTab</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="delimiterLabel">
         <property name="text">
          <string>Delimiter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="delimiterEdit">
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../property.qrc"/>
 </resources>
 <connections/>
</ui>
